backend/ops_api/ops/document/api.py

```diff
@@ -0,0 +1,59 @@
+from flask import Blueprint, Response, current_app, request
+
+from ops_api.ops.document.document_gateway import DocumentGateway
+from ops_api.ops.document.exceptions import ValidationError
+from ops_api.ops.document.schema import DocumentRequestSchema, DocumentResponseSchema, GetDocumentRequestSchema
+from ops_api.ops.document.service import DocumentService
+from ops_api.ops.utils.response import make_response_with_headers
+
+# Create Blueprint object
+documents_bp = Blueprint("documents", __name__)
+
+
+@documents_bp.route("/documents/", methods=["GET"])
+def get_documents_by_agreement_id():
+
+    # Validate request data
+    request_schema = GetDocumentRequestSchema()
+    try:
+        request_data = request_schema.dump(request_schema.load(request.args))
+        request_schema.validate(request_data)
```

Comment:
It should be possible to remove `request_schema.validate` here - `request_schema.load` will call `validate()` during its invocation.  Also the try block can be removed - `backend/ops_api/ops/error_handlers.py` will catch the `ValidationError`